hypixel-api-example/src/main/java/net/hypixel/api/example/GetGuildExample.java

```diff
@@ -36,17 +36,17 @@ public static void main(String[] args) {
         try {
             /*
              * We'll be fetching the guild's stats using its ID for this example, but guilds can
-             * also be looked up using their name, or one of their members' Minecraft UUIDs.
+             * also be looked up by their name, or one of their member's Minecraft UUIDs.
              *  - HypixelAPI.getGuildByName(String)
              *  - HypixelAPI.getGuildByPlayer(UUID)
              */
             String guildId = ExampleUtil.GUILD_ID;
 
             /*
-             * Here, we store the response from the API in our variable.
+             * Here, we store the response from the API.
              *
              * We call `.get()` at the end so that we can use the reply in the same thread.
-             * The downside is that the current thread freezes (or "blocks") until the API responds.
+             * The downside is that this is synchronous operation.
```

Comment:
Original message should be kept as we are here losing the fact that the performances are dependant of the API response time.

Reply:
The current thread being blocked until something happening (in this case the API responding) is the definition of synchronous operation(which dependent the time the API takes to respond is how long the thread takes to be unblocked hence the performance being dependent on api response time)